packages/bitcore-client/bin/wallet-storage

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('../ts_build/program');
+const fs = require('fs');
+const stream = require('stream');
+const { Wallet } = require('../ts_build/wallet');
+const { Storage } = require('../ts_build/storage');
+const { StreamUtil } = require('../ts_build/stream-util');
+
+program
+  .version(require('../package.json').version)
+  .option('--name [name]', 'REQUIRED - Wallet Name')
+  .option('--path [path]', 'optional - Custom wallet storage path')
+  .option('--storageType [storageType]', 'optional - name of the database to use (Mongo or Level)')
+  .option('--exp [exp]', 'export an existing wallet into an encrypted json')
```

Comment:
I would prefer the full word for exp and imp. You can add a short tag, like below if you want it to be shorter.
```
'-e, --export'
'-i, --import'
```